src/main/resources/jooq-configuration.xml

```diff
@@ -18,7 +18,7 @@
         annotation |
         user |
         machine_annotation_services |
-        mas_job_record
+        mas_job_record_new
```

Comment:
Can we do this without the new. Something like:
- first code review
- PR approved
- Fix table to new layout
- Rerun JOOQ and make last fixes
- Merge
- Deploy
Downtime shouldn't be that long and it is just test for acc we could do it right the first time as this hasn't been to acc yet.

Reply:
Sounds good, really didn't like making the `new_` table.